(docs) Contibuting.md Added docsify note, and guidance on snippets (#3056)
felangel/bloc: CONTRIBUTING.md

If you want to help maintain translations in the future, add yourself to `.github/DOCS_ISSUE_TEMPLATE.md`.

> ### Style suggestions

Reviewer: ```suggestion
### Tips
```

> We are grateful for whatever help we can get. Here are a few tips:

Reviewer: ```suggestion
Here are a few tips when contributing to the documentation:
```

> - We use docsfiy for document rendering. See that syntax for ways to include code fragments.

Reviewer: ```suggestion
- We use [docsfiy](https://docsify.js.org) for document rendering. Refer to the docsify documentation for more context/examples regarding the syntax, usage, and capabilities.
```

> - We prefer includes to code files rather than generating new snippets and including. (Some old examples make snippets, which we are slowly migrating.)

Reviewer: ```suggestion
- We prefer to reference real files in GitHub rather than creating snippets. (Some old examples make use of snippets but we are slowly migrating away.)
```

> - We prefer to reference real files in GitHub rather than creating snippets. (Some old examples make use of snippets but we are slowly migrating away.)

Reviewer: ```suggestion
- We prefer to reference real files in GitHub rather than creating snippets (some old examples make use of snippets but we are slowly migrating away).
```
